guides/v2.2/cloud/project/project-upgrade.md

```diff
@@ -1,259 +1,199 @@
 ---
 group: cloud
-title: Upgrade Magento Commerce (Cloud)
+title: Upgrade Magento version
 version: 2.2
 github_link: cloud/project/project-upgrade.md
+functional_areas:
+  - Cloud
+  - Upgrade
 ---
 
-This information details how to upgrade {{site.data.var.ece}} to 2.2.X from any version, and upgrading to new 2.2.X versions.
+You can upgrade the core {{site.data.var.ee}} code base to version 2.2 from any of the following supported versions:
 
-When you upgrade {{site.data.var.ece}}, you also upgrade with patches and available hotfixes as part of the `magento-cloud-metapackage`. Make sure that your project root folder contains an `auth.json` file for authentication.
+-  2.0.14 and later
+-  2.1.7 and later
 
-Our upgrades are Composer driven. For more information on Composer, see [Composer in Cloud]({{ page.baseurl }}/cloud/reference/cloud-composer.html).
+If you need to upgrade from an older version than listed, you must upgrade to a supported version first.
 
-When upgrading from 2.0.X or 2.1.X to 2.2.X, please see [Upgrade from 2.0.X or 2.1.X](#old-version). For additional information on the release, see the [{{site.data.var.ece}} 2.2 Release Notes].
+## Prepare the environment
 
-<div class="bs-callout bs-callout-warning" markdown="1">
-Always apply and test a patch your local system in an active branch. You can push and test in an Integration environment prior to deploying across all environments.
-</div>
+Upgrading from **2.0.X** requires the following tasks:
 
-## Supported upgrade paths to 2.2 {#upgradepaths}
-You can directly upgrade to {{site.data.var.ece}} 2.2 from the following versions:
+-  Upgrade your PHP version to 7.0 or later
+-  Update the `.magento.app.yaml` file with new settings for hooks and environment variables
+-  Verify or set the `ADMIN_EMAIL` variable
+-  Upgrade to the latest supported version of Fastly
 
-* 2.0.X versions: 2.0.14, 2.0.15, 2.0.16
-* 2.1.X versions: 2.1.7, 2.1.8, 2.1.9
+Upgrading from **2.1.X** requires the following tasks:
 
-We have heavily tested and verified upgrades to 2.2 from the latest three versions of 2.0.X and 2.1.X.
+-  Upgrade your PHP version to 7.0 or later
+-  Create a new `config.php` file
+-  Update the `.magento.app.yaml` file with new settings for hooks and environment variables
+-  Verify or set the `ADMIN_EMAIL` variable
+-  Upgrade to the latest supported version of Fastly
+-  Update the `.gitignore` file for new, generated directory
```

Comment:
You're referring to a new directory called `generated`, but that's not 100% clear the way it's written.

> Add the new `generated` directory to the `.gitignore` file